server/src/graphql/schema.graphql

```diff
@@ -17,16 +17,19 @@ schema {
 type Query {
   # Returns the logged-in user making the query, or null if anonymous.
   self: User
-  usersAtTable(tableId: Int!): [User!]!
+  usersAtTable(tableId: Int!): [TableParticipant!]!
```

Comment:
why was this changed? `TableParticipants` are users, better not to prevent multiple data types that accomplish the same thing, and `User` is more generic and flexible

Reply:
fixed